2.2.3/bullseye/include/entrypoint

```diff
@@ -1,5 +1,58 @@
 #!/usr/bin/env bash
 set -e
+set -x
+
+
+# Optional sidecar logging keepalive support
+if [[ -n ${SIDECAR_LOGGING_KEEPALIVE_PORT+x} ]] && [[ "$COMPONENT" =~ ^(scheduler|webserver|worker)$ ]] && [[ $1 != "python" ]]; then
```

Comment:
Why +x here?

Reply:
That's an old suggested syntax that we should not be using. https://github.com/koalaman/shellcheck/wiki/SC2268

Reply:
wait, no. that's a variable substitution that says "If SIDECAR_LOGGING_KEEPALIVE_PORT is defined, print x." i'm not sure why we would use that here. in any case i will fix it.